src/doc/en/api/class/HttpService.md

```diff
@@ -1,4 +1,53 @@
+# Summary
+Allows a server to make HTTP requests.
+
 # Details
+The URL schemes allowed by HttpService are `http` and `https`. Domains are
+filtered using a blacklist. Generally, requests to any URL are allowed, while
+requests to Roblox domains are disallowed. If a URL is disallowed or otherwise
+invalid, then the following error will be thrown:
+
+	URL: Trust check failed
+
+Where `URL` is the given URL.
+
+If the URL is empty, then the following error will be thrown:
+
+	Empty URL
+
 There is a limit to the number of concurrent connections made by HttpService.
 This amount is subject to change, and so should not be relied upon. At least one
 connection at a time is guaranteed.
+
```

Comment:
Check if this limit is per GetAsync+PostAsync+RequestAsync, or global (e.g. ContentProvider, HttpRbxApiService, etc).

Reply:
Behavior appears to have been modified: Instead if throwing, requests will block until there are few enough concurrent connections.